### ISSUE TITLE
Broken Wheel for py3 - 1.6.13

### ISSUE THREAD

**Issue by canni** [jazzband/django-pipeline#647]
A regression of: #616 pulling backport py2 futures

```shell
$ python3 -m venv .venv
$ source .venv/bin/activate
$ python --version
Python 3.6.3
$ pip install django-pipeline
Collecting django-pipeline
  Using cached django_pipeline-1.6.13-py2.py3-none-any.whl
Collecting futures>=2.1.3 (from django-pipeline)
  Using cached futures-3.1.1.tar.gz
Installing collected packages: futures, django-pipeline
  Running setup.py install for futures ... done
Successfully installed django-pipeline-1.6.13 futures-3.1.1
```

Probably there should be a test build checking for that regression

**Comment by rsebille** [comment]
Hello,

The wheel was broken before but this wasn't causing problems, since the release of `futures` 3.2.0 (and this commit https://github.com/agronholm/pythonfutures/commit/d0393ad626d25622927bb0ed47d35ddb2f6cd321) `django-pipeline` can not be installed on Python 3+ anymore.

Because of this dependency, I think django-pipeline shouldn't be an universal wheel, this way we have a proper wheel and dependencies for each Python version.

**Comment by xavfernandez** [comment]
Hello #648 should fix this.

**Comment by EvaSDK** [comment]
As suggested on futures README, #648 should do the work.
https://github.com/agronholm/pythonfutures/blob/master/README.rst

**Comment by EvaSDK** [comment]
Pushed a fixed wheel to company's devpi so I could move on to other things.

**Comment by cyberdelia** [comment]
It should be fixed in 1.6.14.
